include/swift/ClangImporter/SwiftAbstractBasicReader.h

```diff
@@ -0,0 +1,86 @@
+//===- SwiftAbstractBasicReader.h - Clang serialization adapter -*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This file provides an intermediate CRTP class which implements most of
+// Clang's AbstractBasicReader interface, paralleling the behavior defined
+// in SwiftAbstractBasicWriter.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_CLANGIMPORTER_SWIFTABSTRACTBASICREADER_H
+#define SWIFT_CLANGIMPORTER_SWIFTABSTRACTBASICREADER_H
+
+#include "clang/AST/AbstractTypeReader.h"
+// AbstractBasicReader.h doesn't include what it uses
```

Comment:
It is unclear whether this line applies to the header below or above, could you make it an end-of-line-comment instead?

Reply:
Expanded the comment to make it clearer.